test/Main.purs

```diff
@@ -1,51 +1,51 @@
-module Test.Main where

```

Comment:
I...don't really know what's going on with this diff. As far as I can tell nothing changed.

Reply:
I've just ignored these when I've seen them.